common/src/main/java/org/mvndaemon/mvnd/common/logging/TerminalOutput.java

```diff
@@ -767,12 +768,15 @@ private void addProjectLine(final List<AttributedString> lines, Project prj) {
         if (transfer != null) {
             asb
                     .append(':')
+                    .style(CYAN_FOREGROUND)
                     .append(String.format(artifactIdFormat, prj.id))
+                    .style(WHITE_FOREGROUND)
                     .append(transfer);
         } else if (execution == null) {
             asb
                     .append(':')
-                    .append(prj.id);
+                    .style(CYAN_FOREGROUND)
+                    .append(String.format(artifactIdFormat, prj.id));
```

Comment:
I wonder if the project id really needs formatting here.  The main purpose was to trim it to a certain length to keep a nice display.  In this case, there's nothing more on the line though...

Reply:
fixed